.env.example

```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=<postgres_user>
+POSTGRES_PASSWORD=<postgres_password>
```

Comment:
We should be able to get rid of user/password, for the local environment.

Reply:
7692ee685afe3ee1c768dd6ea27d9f2729a00cc8